### DISCUSSION TITLE
Error info during record to postgres table: Cannot read property 'rows' of undefined

### DISCUSSION THREAD

**Discussion by hatjog** [Budibase/budibase#3733] [Help]
During saving data to the postgres table, there is message "Cannot read property 'rows' of undefined",
but data are saved correctly:

![image](https://user-images.githubusercontent.com/536475/145290835-01122fa5-3157-45e1-869a-23a6cf0d9852.png)

**Comment by hatjog** [comment]
Similar to issue: #3031

**Reply by mjashanks** [reply]
Hi @hatjog. I cannot reproduce this in my Postgres DB.

Did anything change in your table in the lead up to this? Have you tried to re-run "Fetch Tables from Database", though I believe that this will clear any relationships that you have set up - so you might want to export your app first (as a backup).

**Reply by hatjog** [reply]
Yes, I added the relationship to this table. 
I did another test: export the app, import app, fetch tables and try do update the record - with the same result - error message.

So I added anothe table from the database, without additional changes in the structure.
And everythong is OK - updates of data are correct, without error messages. 

It seems that cause of the problem is added relationship.

**Comment by mjashanks** [comment]
Would you be able to share `CREATE` scripts for both tables in the relationship?

**Reply by hatjog** [reply]
Yes, 
[sql.txt](https://github.com/Budibase/budibase/files/7687250/sql.txt)
